IPython/testing/iptest.py

```diff
@@ -385,6 +385,12 @@ def run_iptest():
     if '--with-xunit' in sys.argv and not hasattr(Xunit, 'orig_addError'):
         monkeypatch_xunit()
 
+    arg1 = sys.argv[1]
+    if arg1.startswith('IPython/'):
+        if arg1.endswith('.py'):
+            arg1 = arg1[:-3]
+        sys.argv[1] = arg1.replace('/', '.')
```

Comment:
That allows to run iptest with a filepath which is a little more confenient for bash autocompletion.